tests/unittests/libs/test_database.py

```diff
@@ -0,0 +1,34 @@
+from pony import orm as pny
+from qe_metrics.libs.database import JiraIssues, Services
+from tests.unittests.utils import sqlite_table_exists, TEMP_DB_FILE
+
+
+def test_database_initialization(setup_teardown_sqlite_db):
```

Comment:
this is testing pony not the code; please remove; the test fixture should create the db and tables in fixtures. the tests should test the above code.